selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Consoles.java

```diff
@@ -300,4 +304,14 @@ public boolean processesMainAreaIsOpen() {
   public void clickOnMaximizePanelIcon() {
     redrawDriverWait.until(elementToBeClickable(By.xpath(MAXIMIZE_PANEL_ICON))).click();
   }
+
+  /**
+   * * apply drug and drop feature for consoles feature, shift the work bench panel up or down
+   *
+   * @param xoffset offset in pixels for shifting
+   */
+  public void dragConsolesInDefinePosition(int xoffset) {
```

Comment:
If the name of the method is self descriptive I don't want to see any java doc.
Review all javadocs you have added in this PR.  https://dev.liferay.com/participate/javadoc-guidelines

Reply:
The name of the variables `xoffset` seems weird.